### PR TITLE
Environments Setup

### PR DESCRIPTION
**Pull request by luiskcs89**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
     "prepublish": "stencil build --prod",
     "firebase:deploy": "node scripts/deploy.js",
     "firebase:deploy:dev": "firebase use default && firebase deploy --token \"$FIREBASE_TOKEN\"",
+    "firebase:deploy:qa": "firebase use qa && firebase deploy --token \"$FIREBASE_TOKEN\"",
+    "firebase:deploy:staging": "firebase use staging && firebase deploy --token \"$FIREBASE_TOKEN\"",
```
Comment (mm716783) [inline package.json]: Do we still need these scripts here? We are no longer calling them in deploy.js. Maybe we still need them for manual deployments?

**Comment by mm716783** [comment]
Looks good! I suggest adding something like this to readme.md to reflect the updated deployment process:


The branches develop, qa, staging, and master are all set up for continuous integration and deployment with Travis CI and Firebase. To deploy, create a PR for the appropriate branch, or commit and push directly (avoid this if possible).

If manual deployment is needed, use the following. Only do this on the develop branch, follow the PR request flow for all other branches.

```
npm run prepublish
```

_Note:  You may have to install firebase-tools via ```npm install -g firebase-tools``` and then authenticate via command line on your local machine to firebase_

```
npm run firebase:deploy:dev // deploy to firebase dev environment
```

File: package.json (hunk 1 above)
Comment (luiskcs89) [reply package.json]: yes, I am going to leave them in case they are needed for manual deployment.

**Comment by luiskcs89** [comment]
thanks @mm716783 , readme updated

File: package.json (hunk 1 above)
Comment (mm716783) [reply package.json]: Ok sounds good

**Comment by Wholloran** [comment]
Env URL's look good, QA Approved

**Pull request merged**